frontends/web/src/routes/account/utils.ts

```diff
@@ -102,7 +102,13 @@ export type TAccountsByKeystore = {
 
 // Returns the accounts grouped by the keystore fingerprint.
 export function getAccountsByKeystore(accounts: IAccount[]): TAccountsByKeystore[] {
-  return Object.values(accounts.reduce((acc, account) => {
+  const sortedAccounts = accounts
```

Comment:
why don't you sort the `TAccountsByKestore[]` result instead, if the keystore's are out of order? (i assume the accounts inside the keystores are ordered properly).

then you don't have to worry about deep copies either, and not owrry about if the sort is a stable sort and retains the order of the accounts.